templates/compositions/ec-component-content-item/content-item.html.twig

```diff
@@ -0,0 +1,123 @@
+              icon: {}, (associative array) (default: {}): Predefined structure compatible with EC Icon
+            },
+            ...
+          ]
+          links: [], (array) (default: []): List of links (required if Card type is a 'tile'). Format: [
+            {
+              label: (string) (default: ''): Label of link
+              path: (string) (default: ''): Link url (href attribute)
+            },
+            ...
+          ]
+        }
+    - "extra_classes" (string) (default: '')
+    - "extra_attributes" (array) (default: []): format: [
+        {
+          "name" (string) (default: ''),
+          "value" (string) (default: '')
+        },
+        ...
+      ]
+#}
+
+{# Internal properties #}
+
+{% set _content_item = {
+  type: 'default',
+  description: '',
+  meta: [],
+  title: {},
+  image: '',
+  image_position: 'before',
+  infos: [],
+  tags: [],
+  links: []
+} %}
+
+{% if content_item is defined %}
+  {% set _content_item = _content_item|merge(content_item) %}
+{% endif %}
+
+{% set _css_class = 'ecl-content-item ecl-u-d-flex ecl-u-border-bottom ecl-u-border-color-grey-15 ecl-u-pv-m' %}
+{% set _extra_attributes = '' %}
+
+{# Internal logic - Process properties #}
+
+{% if extra_classes is defined and extra_classes is not empty %}
+  {% set _css_class = _css_class ~ ' ' ~ extra_classes %}
+{% endif %}
+
+{% if extra_attributes is defined and extra_attributes is not empty and extra_attributes is iterable %}
+  {% for attr in extra_attributes %}
+    {% set _extra_attributes = _extra_attributes ~ ' ' ~ attr.name ~ '="' ~ attr.value ~ '"' %}
+  {% endfor %}
+{% endif %}
+
+{# Print the result #}
+
+<article class="{{ _css_class }}"{{ _extra_attributes|raw }}>
+  {% if _content_item.image and _content_item.image_position == 'before' %}
+    <div role="img" aria-label="{{ _content_item.image.alt }}" class="ecl-u-ratio-3-2 ecl-u-flex-shrink-0 ecl-u-mr-xl ecl-u-d-lg-block"
+         style="background-image:url(&quot;{{ _content_item.image.src }}&quot;);background-size:contain;height:5rem;width:7.5rem"></div>
+    <div role="img" aria-label="{{ _content_item.image.alt }}"
+         class="ecl-u-ratio-3-2 ecl-u-flex-shrink-0 ecl-u-mr-xl ecl-u-d-none ecl-u-d-lg-none"
+         style="background-image:url(&quot;{{ _content_item.image.src }}&quot;);background-size:contain;height:8.75rem;width:13.125rem">
+    </div>
+  {% endif %}
+  <div class="ecl-u-flex-grow-1">
+    {% if _content_item.meta is not empty %}
+      <div class="ecl-content-item__meta ecl-u-type-s ecl-u-type-color-grey-75">{{ _content_item.meta|join(" | ") }}</div>
+    {% endif %}
+    <div class="ecl-content-item__title ecl-u-type-prolonged-m ecl-u-type-bold ecl-u-mt-xs">
+      {%- if _content_item.title.path %}
+        {% include '../ec-component-link/link.html.twig' with {
+          link: _content_item.title
+        } only %}
+      {% else %}
+        {{- _content_item.title.label }}
+      {%- endif -%}
```

Comment:
The HTML on [the playground](https://ec.europa.eu/component-library/playground/ec/?path=/story/templates-compositions--content-items) does not use the link component right? But a simple `a` tag. Visuals seems to be different, like this implementation has the title underlined while on the preview it looks like it's not:

![image](https://user-images.githubusercontent.com/153362/62692659-5e207680-b9d1-11e9-954a-eca81d48d28f.png)

Reply:
Good catch. The title is a link with the type 'standalone'. updated.